codec-http2/src/main/java/io/netty/handler/codec/http2/Http2StreamFrame.java

```diff
@@ -23,15 +23,19 @@
  * cases, the {@link #streamId()} must return {@code 0}. If the frame applies to a stream, the
  * {@link #streamId()} must be greater than zero.
  */
+//TODO(buchgr): Do we REALLY need the flexibility of supporting stream id 0? It seems confusing.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)